.github/workflows/commit-message.yml

```
@@ -22,7 +22,7 @@ jobs:
           flags: 'gm'
           excludeDescription: 'true' # optional: this excludes the description body of a pull request
           excludeTitle: 'true' # optional: this excludes the title of a pull request
-          error: 'The first line has to start with the type (e.g "ENH: Add support for awesome feature"). For more details, see https://slicer.readthedocs.io/en/latest/developer_guide/style_guide.html#commits'
+          error: 'The first line has to start with the type followed by a message with a capital letter (e.g "ENH: Add support for awesome feature"). For more details on other requirements, see https://slicer.readthedocs.io/en/latest/developer_guide/style_guide.html#commits'
```

Let’s be a little more specific about what “the type” is referring to. I can see others not knowing this is referring to a commit prefix and that there is also a colon and space needed. A regex can be at times confusing to read. 
```suggestion
          error: 'The first line has to start with a commit prefix, followed by a colon and space, and then followed by a message with a capital letter (e.g "ENH: Add support for awesome feature"). For more details on other requirements, see https://slicer.readthedocs.io/en/latest/developer_guide/style_guide.html#commits'
```

---

Note that I don’t know if it’s clearly understood that the colon or space is included in the commit message prefix. So I wrote it as though the prefix is simply the “ENH” part